fe/fe-core/src/main/java/com/starrocks/mysql/MysqlProto.java

```diff
@@ -394,6 +405,14 @@ public static byte[] readNulTerminateString(ByteBuffer buffer) {
         return buf;
     }
 
+    public static boolean isRemoteIPLocalhost(String remoteIP) {
+        if (remoteIP == null) {
+            return false;
+        }
+        //Using "String.contains" here because the remote IP address starts with a forward slash, like “/127.0.0.1”.
+        return remoteIP.contains(LOCALHOST);
```

Comment:
not sure if the IPv6 localhost `::1` should be allowed too, FE supports IPv6 stack.

Reply:
@kevincai, discussed about IPv6 support here: https://github.com/StarRocks/starrocks/pull/56176#discussion_r1980745842

Reply:
thanks.